src/valkey-benchmark.c

```diff
@@ -1460,6 +1413,19 @@ int parseOptions(int argc, char **argv) {
                 config.num_threads = 0;
         } else if (!strcmp(argv[i], "--cluster")) {
             config.cluster_mode = 1;
+        } else if (!strcmp(argv[i], "--rfr")) {
+            config.read_from_replica = FROM_REPLICA_ONLY;
+            if (argv[i+1]) {
+                if (!strcmp(argv[i+1], "all")) {
+                    config.read_from_replica = FROM_ALL;
+                    i++;
+                } else if (!strcmp(argv[i+1], "replicas")) {
+                    config.read_from_replica = FROM_REPLICA_ONLY;
+                    i++;
+                } else if (strlen(argv[i+1]) > 0 && argv[i+1][0] != '-') {
+                    goto invalid;
+                }
+            }
```

Comment:
```suggestion
        } else if (!strcmp(argv[i], "--rfr")) {
            if (argv[++i]) {
                if (!strcmp(argv[i], "all")) {
                    config.read_from_replica = FROM_ALL;
                } else if (!strcmp(argv[i], "yes")) {
                    config.read_from_replica = FROM_REPLICA_ONLY;
                } else if (!strcmp(argv[i], "no")) {
                    config.read_from_replica = FROM_PRIMARY_ONLY;
                } else {
                    goto invalid;
                }
            } else
                goto invalid;
```

Reply:
In this code, If we have other options after '--rfr', the other option can't execute, right?

e.g. `--rfr -r 10`

Reply:
it makes it clearer IMO

Reply:
1 more question : Do we need the lines at 1427 ~ 1428?
This seems to mean that if no mode is specified after the `--rfr` option, it will fail to execute. Is it intended that --rfr must be used with mode?

Reply:
IMO yes. having the same option which can either accept a value or not is something I rarely seen. I think it is clearer that a value is either expected or not.